src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { setQuery, setStatus, clearFilter } from '../../features/filter';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const status = useAppSelector(state => state.filter.status);
+  const query = useAppSelector(state => state.filter.query);
```

Comment:
```suggestion
  const { status, query } = useAppSelector(state => state.filter);
```
nit: You can get 2 properties with one selector